.happo.js

```diff
@@ -0,0 +1,18 @@
+const { RemoteBrowserTarget } = require('happo.io')
+
+module.exports = {
+  apiKey: process.env.HAPPO_API_KEY,
+  apiSecret: process.env.HAPPO_API_SECRET,
+  targets: {
+    'Chrome - Desktop': new RemoteBrowserTarget('chrome', {
+      viewport: '1024x768',
+    }),
+    'Firefox - Desktop': new RemoteBrowserTarget('firefox', {
+      viewport: '1024x768',
+    }),
+    'Chrome - Mobile': new RemoteBrowserTarget('chrome', {
+      viewport: '320x640',
+    }),
+    'Safari - Mobile': new RemoteBrowserTarget('ios-safari'),
```

Comment:
This will need a `viewport` as well for the build to succeed. You can add any width and height, it will just ignore it. E.g. `viewport: '375x700'`

Reply:
I see, I'll add a viewport here! Thanks.